db/migrate/20191120152056_add_global_activity_analytics.rb

```diff
@@ -0,0 +1,19 @@
+class AddGlobalActivityAnalytics < ActiveRecord::Migration[5.2]
+  def up
+    execute <<~SQL
+      create table global_activity_analytics (
+        id integer primary key default 1,
+        users integer default 0 not null,
+        countries integer default 0 not null,
+        launches integer default 0 not null,
+        gospel_presentations integer default 0 not null,
+        updated_at timestamp without time zone not null,
+        constraint singleton check (id = 1)
+      )
+    SQL
```

Comment:
Can this be done with ruby code?  It looks like there's a `singleton_guard` option that might work.